includes/admin_import.php

```diff
@@ -765,6 +765,12 @@ function tmsl_import_page() {
 										<input type="submit" class="button" value="<?php _e('Add', '12-step-meeting-list')?>">
 									</div>
 								</form>
+							<?php } else {?>
+								<details>
+									<summary><strong><?php _e('Public Feed', '12-step-meeting-list')?></strong></summary>
+									<p><?php _e('The following feed contains your publicly available meeting information.')?></p>
```

Comment:
We need the string domain on here so it's picked up for translation https://developer.wordpress.org/reference/functions/_e/

`information.', '12-step-meeting-list')?>`

Reply:
Procedural question: Does @brianw-area24 need to add this into the code and do another PR? Or, will the PR update?

Reply:
He should be able to commit on his branch to update this PR

Reply:
(or one of us could)

Reply:
Yes, I pushed a new commit up to my branch on my fork of the plugin, and it automagically changed the PR to include my new commit.

I need to study up on translations, I've not worked with those before. Thanks for pointing out my omission.

Reply:
When we change/add a string like I did, do we need to modify language/12-step-meeting-list.pot, or is there a process that will do that for us?

Reply:
I'm honestly not sure, it may be that `12-step-meeting-list.pot` needs to be updated manually, although I haven't been doing that. My understanding is that adding the domains like this keep translate.wordpress.org up to date.